build/generic/web/viewer.css

```diff
@@ -664,6 +673,43 @@ i.svg-icon g {
   background: rgba(0, 0, 0, 0.2);
 }
 
+#mozPrintCallback-shim {
+  border-radius: 4px;
+}
+
+/* Display a fixed page number overlay on very small devices. */
+@media all and (max-width: 600px) {
+   #lightTheme #pageNumberWrapper {
+      position: fixed;
+      top: 65px;
+      left: 0;
+      right: 0;
+      text-align: center;
+   }
+
+   #lightTheme #pageNumberInfo {
+      display: inline-block;
+      min-width: 60px;
+      padding: 6px 8px;
+      border-radius: 4px;
+      background: rgba(0, 3, 6, 0.61);
+      color: #FAFAFA;
+      font-weight: 600;
+      letter-spacing: -0.29px;
+      transition: 0.3s opacity;
+      -webkit-transition: 0.3s opacity;
+      opacity: 1;
+   }
+
+   #lightTheme #pageNumberInfo span {
+      color: inherit;
+   }
+
+   #lightTheme #pageNumberInfo.fadeOut {
+      opacity: 0;
```

Comment:
I tried `transition-delay`—the problem was I still had to do a short `setTimeout` to "cancel" and "restart" the animation on `'scroll'`. So I thought it would be simpler to just use `setTimeout`, rather than both.